.env

```diff
@@ -0,0 +1 @@
+PORT=3001
```

Comment:
I think this file should be in .gitignore

Reply:
well, when it was there i was having a lot of problems
this is why i took it out...mr google told me it was a good solution and i think it worked